src/DbInterface.cpp

```diff
@@ -620,6 +620,7 @@ void DbInterface::getServerIpAddress(std::shared_ptr<swss::DBConnector> configDb
     std::vector<swss::KeyOpFieldsValuesTuple> entries;
 
     configDbMuxCableTable.getContent(entries);
+    mMuxManagerPtr->updateWarmRestartReconciliationCount(entries.size());
```

Comment:
At this stage linkmgrd won't be able to tell which one are unhealthy, it will probe all ports' mux state. The size of `entries` is the number of ports configured in `MUX_CABLE|PORTNAME` tables.